R/connector.R

```diff
@@ -26,7 +26,11 @@ data_connector <- function(date = "latest"){
       attr(fun, "table") <- .x
       fun
     }) %>%
-    set_names(list_tables_s3())
+    set_names(
```

Comment:
This works, but shouldn't the change happen earlier in the pipeline? That is, shouldn't the tables be given different names?

Reply:
So I am doing this here so that even if we change the table names in our pipeline to `course_fact` etc, it will still work.

Reply:
Do it need to change in `list_tables_s3()` too? It would be weird to list tables and see different names.